### DISCUSSION TITLE
What other key are in there in items of collections.tag?

### DISCUSSION THREAD

**Discussion by avinashkanaujiya** [11ty/eleventy#3115] [Q&A]
```
{% for item in collections.posts %} {% if item.data.publish === "true" %}
<div>
  {{ item.data.publish }}
  <a href="{{item.page.url}}">{{ item.page.fileSlug}}</a>
</div>
<!-- code to render if the condition is true -->
{% elif another_condition %}
<!-- code to render if the another_condition is true -->
{% else %}
<!-- code to render if none of the conditions are true -->
{% endif %} {% endfor %}
```

Other than data and page what are available and if I want to see , how do I go about doing that?

**Comment by Snapstromegon** [comment]
The answer you're looking for is documented here: https://www.11ty.dev/docs/collections/#collection-item-data-structure

To put it in a nutshell:
- page: The page object itself
- data: The data of the page
- content: The content of the rendered page without layout wrappers

In addition to that there are still some more keys for backwards compatibility, which are replaced by things above:
- inputPath
- fileSlug
- outputPath
- url
- date
- templateContent
